docs/guidelines/content-guidelines/03-formatting.md

```diff
@@ -80,11 +80,12 @@ Suppose that it is only the part containing the certificate information that is
 >**NOTE** The following panels are not rendered correctly when indented. Use them only in the docs that are displayed on the website.
 For the docs not rendered on the website, use **NOTE**, **TIP**, and **CAUTION**.
 
-Panels are colorful containers that call out important or additional information within a topic. The Kyma documentation uses the [Flexible Alerts](https://github.com/fzankl/docsify-plugin-flexible-alerts) docsify plugin for the panels' formatting. To call attention to a specific note, a word of caution or a tip, use the  [!NOTE], [!TIP], and [!WARNING] panels. Use:
+Panels are colorful containers that call out important or additional information within a topic. The Kyma documentation uses the VitePress customized alerts. To call attention to a specific note, a word of caution or a tip, use the  [!NOTE], [!TIP], [!WARNING], and [!IMPORTANT] panels. Use:
 
 - The blue [!NOTE] panel to point to something specific, usually relating to the topic.
 - The orange [!WARNING] panel to call attention to something critical that can cause inoperable behavior.
 - The green [!TIP] panel to share helpful advice, such as a shortcut to save time.
+- The purple [!IMPORTANT] panel to mention something a user must be aware of.
```

Comment:
I'm against using this label. There's no corresponding label in DITA (so we'd have trouble keeping docs in sync) and I don't see how it adds value compared to the other three types. If there is any note/warning/tip should that isn't important, we shouldn't include it at all.